cli/pack/app_dir.go

```diff
@@ -336,6 +337,15 @@ func copyProjectFiles(dst string, ctx *context.Ctx) error {
 				return false, fmt.Errorf("Failed to get file rel path: %s", err)
 			}
 
+			// We need to ignore the run_dir, data_dir and log_dir folders because it
+			// contains the files of the running instance, which can then block application pack.
+			// More details: https://github.com/tarantool/cartridge-cli/issues/494
+			if strings.HasPrefix(relPath, project.DefaultLocalDataDir) ||
```

Comment:
Why can't we get these values from the `Running context` (before calling the function `SetLocalRunningPaths`)?
AFAUI in this case we will also take into account the value from the config and `defaultLocal...Dir`will remain private as they were.

Reply:
If you talk about ``ctx.Running.RunDir`` (and others), they point to the wrong directories. For example ctx.Running.RunDir points to `/var/run/tarantool`. This is set to into `SetSystemRunningPaths` function (`SetLocalRunningPaths` function isn't called wheb pack processing). Before `SetSystemRunningPaths` call, this fields is empty.

Reply:
>`SetLocalRunningPaths` function isn't called when pack processing
Yep, but why can't we start doing this? In this case, `ctx.Running.RunDir` will contain the actual value.

Reply:
Agree, change `SetSystemRunningPaths` -> `SetLocalRunningPaths`

Reply:
In fact, we cannot make this replacement. Some of the paths that are formed in SetSystemRunningPaths are needed during the packaging process. I suggest leaving the changes as they are now (or just ignore the ``tmp/`` folder)

Reply:
Wow, how difficult it is. I will try to propose yet one idea. We can add to `files.go` a new function `GetLocalRunningPaths` which will return all "local" paths, call it inside `copyProjectFiles` and exclude the corresponding paths.

Reply:
Added.